app/controllers/api/storage.php

```diff
@@ -381,6 +403,8 @@
             throw new Exception('Bucket not found', 404);
         }
 
+        $read ??= $bucket->getAttribute('$read', []); // By default inherit read permissions
+        $write ??= $bucket->getAttribute('$write', []); // By default inherit write permissions
         $read ??= $bucket->getAttribute('$read', []); // By default inherit read permissions
         $write ??= $bucket->getAttribute('$write',[]); // By default inherit write permissions
```

Comment:
This looks...off to me. Can we get rid of the duplicate lines here?

Reply:
Oh wow, how did this happen.